test/cases/regression-include-excluded-packages.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+# This test case verifies that a blueprint can include a package which
+# is listed among "excluded" for a certain image type and osbuild-composer
+# doesn't fail to depsolve this blueprint.
+#
+# The script currently works only for RHEL and CentOS which provide
+# "redhat-lsb-core" package and exclude "nss" package in the image type
+# definition. The testing blueprint contains explicit "nss" requirement
+# to remove it from the list of excluded packages and thus enable the
+# installation of "redhat-lsb-core".
+#
+# Bug report: https://github.com/osbuild/osbuild-composer/issues/921
+
+set -xeuo pipefail
+
+# Provision the software under tet.
+/usr/libexec/osbuild-composer-test/provision.sh
```

Comment:
Won't that conflict/fail b/c regression.sh also calls provision.sh ? I can see for example:
```
sudo systemctl start osbuild-remote-worker.socket
sudo systemctl start osbuild-composer.socket
sudo systemctl start osbuild-composer-api.socket
```

which should probably be a restart or an if condition should be placed in provision.sh.